### ISSUE TITLE
HWK 290 title slot scum and villany

### ISSUE THREAD

**Issue by Jedi34** [Sandrem/FlyCasual#3460]
The title slot and the moldy crow upgrade are missing in the Scum and Villany Faction but there are still in the Rebel Alliance Faction
![image](https://user-images.githubusercontent.com/93074364/160884882-67dd26da-2e0f-48f4-8c55-b9b000b5336d.png)
![image](https://user-images.githubusercontent.com/93074364/160885162-44c5ddcd-f3d8-4f01-9df6-17192a98d054.png)

**Comment by grandpajive** [comment]
Right. Only Palob in scum can take it, the rest can't. Torkil is sad.

**Comment by Jedi34** [comment]
Thank you very much for your answer
